CHANGELOG.md

```diff
@@ -1001,7 +1001,7 @@ createTestCafe('localhost', 1337, 1338)
             .run();
     })
     .then(() => {
-        testcafe.close();
```

Comment:
I think we should use `async/await` in all examples. Like here:
https://github.com/DevExpress/testcafe/pull/5234/files#diff-97f23936b8a1a07bba3fa957612231adR27